library_schemas/score/prerelease/HED_score_1.1.1.mediawiki

```diff
@@ -12,953 +12,587 @@ For more information see https://hed-schema-library.readthedocs.io/en/latest/ind
+    * Non-reactive-sleep <nowiki>[EEG activity consisting of normal sleep graphoelements, but which cannot be interrupted by external stimuli/ the patient cannot be waken. (Source: Beniczky ea 2013, Appendix S3; Beniczky ea 2017, Section 7.)]</nowiki>
+
+
+'''Interictal-finding''' <nowiki> [EEG pattern / transient that is distinguished form the background activity, considered abnormal, but is not recorded during ictal period (seizure) or postictal period; the presence of an interictal finding does not necessarily imply that the patient has epilepsy. (Source: Beniczky ea 2013, Appendix S1; Beniczky ea 2017, Table 5.)]</nowiki>
+
+    * Epileptiform-interictal-activity <nowiki> {suggestedTag=Spike-morphology, suggestedTag=Spike-and-slow-wave-morphology, suggestedTag=Runs-of-rapid-spikes-morphology, suggestedTag=Polyspikes-morphology, suggestedTag=Polyspike-and-slow-wave-morphology, suggestedTag=Sharp-wave-morphology, suggestedTag=Sharp-and-slow-wave-morphology, suggestedTag=Slow-sharp-wave-morphology, suggestedTag=High-frequency-oscillation-morphology, suggestedTag=Hypsarrhythmia-classic-morphology, suggestedTag=Hypsarrhythmia-modified-morphology, suggestedTag=Categorical-location-value, suggestedTag=Sensors, suggestedTag=Finding-propagation, suggestedTag=Multifocal-finding, suggestedTag=Appearance-mode, suggestedTag=Discharge-pattern, suggestedTag=Finding-incidence} [Transients distinguishable from background activity, with characteristic spiky morphology, typically, but neither exclusively, nor invariably found in interictal EEGs of people with epilepsy. (Source: Beniczky ea 2013, Appendix S4; Morphologies from Beniczky ea 2017, Table 5; Suggested tags from Beniczky ea 2017, Section 8.)]</nowiki>
+    * Abnormal-interictal-rhythmic-activity <nowiki> {suggestedTag=Rhythmic-activity-attribute, suggestedTag=Polymorphic-delta-activity-morphology, suggestedTag=Frontal-intermittent-rhythmic-delta-activity-morphology, suggestedTag=Occipital-intermittent-rhythmic-delta-activity-morphology, suggestedTag=Temporal-intermittent-rhythmic-delta-activity-morphology, suggestedTag=Categorical-location-value, suggestedTag=Sensors, suggestedTag=Appearance-mode, suggestedTag=Discharge-pattern, suggestedTag=Finding-incidence}[Activity of frequency lower than alpha, that clearly exceeds the amount considered physiologically normal for the patients age and state of alertness. (Source: Beniczky ea 2013, Appendix S4; Morphologies from Beniczky ea 2017, Table 5; Suggested tags from Beniczky ea 2017, Section 8.)] </nowiki>
```

Comment:
for the patients age  --> for patient age

Reply:
fixed